src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -760,7 +761,7 @@ public virtual TextInfo TextInfo
             }
         }
 
-        public override bool Equals(object value)
+        public override bool Equals(object? value)
```

Comment:
GitHub won't let me comment on it, but I think a bunch of the virtuals above are wrong, in that there's nothing preventing a derived type from returning null, e.g. this compiles fine and prints out `true`:
```C#
using System;
using System.Globalization;

    class Program
    {
        static void Main() => Console.WriteLine(new MyCultureInfo().DisplayName == null);
    }

class MyCultureInfo : CultureInfo
{
    public MyCultureInfo() : base("en-US") { }
    public override string DisplayName => null;
}
```

@MadsTorgersen, @terrajobst, how should we be thinking about this.  If the annotations are meant to reflect what's possible, then most of these should return nullable.  If the annotations are meant to reflect what's intended, then most of these should return non-nullable, and derived types that violate that would need to be fixed.

cc: @danmosemsft

Reply:
I've not fixed this yet until we decide on guidelines - I personally lean toward what makes sense vs what is possible - so i.e. would not expect CompareInfo and other similar to be null so IMO should be non-nullable but technically you could do what you described above.

Reply:
I feel strongly that we should annotate intent. That's what the feature is for. It puts the onus on misbehaving implementations, not innocent users.